presentation/src/main/java/com/android254/presentation/common/components/ImageAvater.kt

```diff
@@ -0,0 +1,2 @@
+package com.android254.presentation.common.components
```

Comment:
File naming is off

Change to `ImageAvatar`

Reply:
Fixed